openfisca_us/variables/gov/states/va/tax/income/adjusted_gross_income/va_adjusted_gross_income.py

```diff
@@ -0,0 +1,27 @@
+from re import sub
+from openfisca_us.model_api import *
+
+
+class va_adjusted_gross_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "VA adjusted gross income"
+    unit = USD
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        agi = tax_unit("adjusted_gross_income", period)
+        additions = tax_unit("va_income_additions", period)
+        age_deduction = tax_unit("va_age_deduction", period)
```

Comment:
Could you make a parameter listing the deductions? Then you can do this:
```suggestion
        deductions = add(tax_unit, period, parameters(period).gov.states.va.[...].deductions)
```
followed by
```
return agi + additions - deductions
```